### DISCUSSION TITLE
"crown area" vs. "canopy area"

### DISCUSSION THREAD

**Discussion by adamhb** [NGEET/fates#907] [Q&A] [answered]
I'm trying to figure out the difference between crown area and canopy area in the history variables. Is FATES_CROWNAREA_PF like foliar projective cover where it's the vertical projection of all the tree crowns onto a horizontal surface ([0,1])?

By comparison can FATES_CANOPYAREA_AP get larger than 1? If so, how is it different to LAI?

Any help clarifying these two variables would be greatly appreciated!
Adam

**Comment by JessicaNeedham** [comment, marked as answer]
Hi @adamhb , 

Good question. 

I believe that FATES_CROWNAREA_PF can be greater than 1 since it is the crown area of multiple canopy layers (canopy and potentially multiple understory cohorts). It is indexed by pft. 

FATES_CANOPYCROWNAREA_PF should be between 0 and 1 since it is only counting canopy cohorts. Also indexed by pft. 

FATES_CANOPYAREA_AP as far as I can tell is the same as FATES_CROWNAREA_PF but indexed by patch age rather than pft. So I think it can be greater than 1. 

I think LAI is different because it includes leaf layers (within canopy layers). 

The naming convention is kind of confusing. I wonder if FATES_CANOPYAREA_AP should be renamed FATES_CROWNAREA_AP?

**Reply by ckoven** [reply]
I agree with you both that this is confusing as it currently stands.  I believe the intention is that "crown area" should refer to the crown area of plants irrespective of what layer they are in, and "canopy" should only refer to the uppermost layer of plants. So I support @JessicaNeedham's suggestion of renaming.  There is still some ambiguity since crown area is defined via allometry at the individual tree level, which is never reported to history, but I think that so long as the long name specifies that crown area refers to all plants that is ok?

**Comment by adamhb** [comment]
OK, thank you @JessicaNeedham and @ckoven. Just so its totally clear, if we are quantifying crown area of just the canopy layer would that be the same as LAI in the top leaf layer of the canopy layer? Or would it be the percentage of ground area occupied by the vertical projection of all leaves in all leaf layers in the canopy layer (i.e. the metric "foliage projective cover")? From @JessicaNeedham's responses it sounds like the former, but just wanted to be sure! Thanks again.
Adam

**Reply by ckoven** [reply]
@adamhb Crown area is different from leaf area.  Crown area is basically the footprint of ground that a given plant covers, and there can be greater or less than one leaf layer within a given plant's crown area. I believe that crown area does correspond to the "foliage projective cover" metric.

**Comment by adamhb** [comment]
@ckoven  Ok, thanks a lot! I agree with @JessicaNeedham's name change proposal and yes, also agree that it would help to specify that this metric is "all tree crowns".
